src/test/javascript/spec/component/modeling-assessment-editor/modeling-assessment-editor.component.spec.ts

```diff
@@ -469,4 +473,23 @@ describe('ModelingAssessmentEditorComponent', () => {
             expect(fake).to.have.been.calledOnce;
         }));
     });
+    it('should invoke import example submission', () => {
+        const course = new Course();
+        component.modelingExercise = new ModelingExercise(UMLDiagramType.ClassDiagram, course, undefined);
+        component.modelingExercise.id = 1;
+        component.submission = {
+            id: 2,
+            submitted: true,
+            type: 'MANUAL',
+            text: 'Test\n\nTest\n\nTest',
+        } as unknown as ModelingSubmission;
```

Comment:
Could you explain to me what this does? Why not just `as ModelingSubmission`?

Reply:
I used similar initialization what we used in the lines above. normally, it sets an unknown type to other type. But I think it can be defined without unknown.